mesonbuild/dependencies/pkgconfig.py

```diff
@@ -256,7 +262,7 @@ def _check_pkgconfig(self, pkgbin: ExternalProgram) -> T.Optional[str]:
     def _get_env(self, uninstalled: bool = False) -> EnvironmentVariables:
         env = EnvironmentVariables()
         key = OptionKey('pkg_config_path', machine=self.for_machine)
-        extra_paths: T.List[str] = self.env.coredata.optstore.get_value(key)[:]
+        extra_paths: T.List[str] = self.env.coredata.optstore.get_value(key)[:] + self.extra_paths
```

Comment:
We don't need the cryptic copy of the first list anymore, right?

Reply:
Ah, yes, thanks for catching that.